apps/server/src/modules/tool/controller/tool.controller.spec.ts

```diff
@@ -67,4 +78,83 @@ describe('ToolController', () => {
 			expect(lti11Uc.getLaunchParameters).toHaveBeenCalledWith(currentUser, toolId, courseId);
 		});
 	});
+
+	describe('createExternalTool', () => {
+		function setup() {
+			const bodyConfigCreateBasicParams = new BasicToolConfigParams();
+			bodyConfigCreateBasicParams.type = ToolConfigType.BASIC;
+			bodyConfigCreateBasicParams.baseUrl = 'mockUrl';
+
+			const bodyConfigCreateLti11Params = new Lti11ToolConfigParams();
```

Comment:
we can use it like this
const bodyConfigCreateLti11Params = plainToClass(Lti11ToolConfigParams, {...});

Reply:
an interesting way of doing it, but it isn't used in project in any tests atm... maybe we'll wait for a complete codestyle with this as requirement ^^